nav2_rviz_plugins/include/nav2_rviz_plugins/lifecycle_status_indicator.hpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2019 Intel Corporation
```

Comment:
I didn't know you worked for Intel last year ;-)

Reply:
Oops! Seems like my copy-paste abilities that gave me that position (sorry about that!)